app/utils/lava_log_parser.py

```diff
@@ -72,11 +75,49 @@ def run(log, boot, txt, html):
     else:
         boot_result_html = "<span class=\"warn\">{}</span>".format(boot_result)
 
+    # panic, error and warn are considered deprecated, but still here
+    # for ensure that everything will works well in any case.
     formats = {
-        "warning": "<span class=\"warn\">{}</span>\n",
+        "emerg": "<span class=\"alert\">{}</span>\n",
+        "alert": "<span class=\"alert\">{}</span>\n",
+        "crit": "<span class=\"alert\">{}</span>\n",
         "error": "<span class=\"err\">{}</span>\n",
+        "warning": "<span class=\"warn\">{}</span>\n",
+        "notice": "<span class=\"info\">{}</span>\n",
+        "info": "<span class=\"info\">{}</span>\n",
+        "debug": "<span class=\"debug\">{}</span>\n"
     }
-    numbers = {"warning": 0, "error": 0}
+
+    kernel_log_levels = {
+        "0": "<span class=\"alert\">{}</span>\n",
+        "1": "<span class=\"alert\">{}</span>\n",
+        "2": "<span class=\"alert\">{}</span>\n",
+        "3": "<span class=\"err\">{}</span>\n",
+        "4": "<span class=\"warn\">{}</span>\n",
+        "5": "<span class=\"info\">{}</span>\n",
+        "6": "<span class=\"info\">{}</span>\n",
+        "7": "<span class=\"debug\">{}</span>\n",
+    }
+
+    numbers = {
+        "emerg": 0,
+        "alert": 0,
+        "crit": 0,
+        "error": 0,
+        "warning": 0,
+        "notice": 0,
+        "info": 0,
+        "debug": 0,
+    }
+
+    kernel_numbers = {
```

Comment:
Still not quite correct yet...

Reply:
I have updated this @gctucker, last doubt, may we print any kernel error like we did for lava log warn and error messages?